pony/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ponylang/ponyc:latest
```

Comment:
I prefer to use a tagger version `0.33.1` ;-) 
This is OK since the doc will show `0.33` (this tag will be updated with a bot, and me to have the same version in results)